src/nvim/ex_docmd.c

```diff
@@ -6861,7 +6905,26 @@ void ex_cd(exarg_T *eap)
     if (new_dir == NULL || vim_chdir(new_dir))
       EMSG(_(e_failed));
     else {
-      post_chdir(eap->cmdidx == CMD_lcd || eap->cmdidx == CMD_lchdir);
+      int scope = -1; // Depends on what command was invoked
```

Comment:
Enum type. I would suggest default it to kScopeEditor and use `default: break;` in `switch`: you are in `ex_cd` function, it *won’t* possibly run with non-chdir command, so no need to assert zero.